cloud/errors.go

```diff
@@ -41,3 +42,63 @@ type VMDeletedError struct {
 func (vde VMDeletedError) Error() string {
 	return fmt.Sprintf("VM with provider id %q has been deleted", vde.ProviderID)
 }
+
+// ReconcileError represents an error that is not automatically recoverable
+// recoveryType indicates what type of action is required to recover. It can take two values
+// 1. `Transient` - Can be recovered through manual intervention, will be requeued after
+// 2. `Terminal` - Cannot be recovered, will not be requeued
+type ReconcileError struct {
+	error
+	recoveryType string
+	requestAfter time.Duration
+}
+
+const (
+	// Transient ...
+	Transient = "Transient"
+	// Terminal ...
+	Terminal = "Terminal"
+)
+
+// Error returns the error message for the ReconcileError
+func (t ReconcileError) Error() string {
+	var errStr string
+	if t.error != nil {
+		errStr = t.error.Error()
+	}
+	switch t.recoveryType {
+	case Transient:
+		return fmt.Sprintf("reconcile error occurred that requires manual intervention. Object will be requeued after %s "+
```

Comment:
```suggestion
		return fmt.Sprintf("reconcile error occurred. Object will be requeued after %s "+
```

Reply:
A transient error should not require manual intervention.

Reply:
Yeah, forgot to remove it. Thanks!